### ISSUE TITLE
How shall we structure the repo?

### ISSUE THREAD

**Issue by Cheukting** [ossf/DevRel-community#3]
Open to discussion regarding how the repo should be structured.

**Comment by edodusi** [comment]
We should open specific issues about subfolders of this repo containing resources in specific areas, for example `guidelines`, `assets`, `FAQ`, etc..

I think this could be an ongoing process, don't see the need to aprioristically structure the repo: instead, this repo will assume a structure because we have decided step by step where to write a document or to place an asset, and this can be improved over time.

**Comment by Cheukting** [comment]
Yes, we could, feel free to take the lead! Maybe we can reference back to this issue as well

**Comment by tabathad** [comment]
closing as stale
